### PR TITLE
New package: atari800

### PR DESCRIPTION
**Pull request by skrzyp**
Here's my first XBPS package template!
It adds an Atari800, powerful and multiplatform Atari 8-bit computers emulator.

Thanks from advance for every tips and guidelines how to improve that port.

### CODE REVIEW

**Comment by bougyman** [comment]
This is failing to cross-compile here. Works find on x86 and x86_64 but on armv7hf I get

```
checking SDL_opengl.h usability... no
checking SDL_opengl.h presence... no
checking for SDL_opengl.h... no
configure: error: unable to use OpenGL  - SDL_opengl.h not found or not compilable.
=> ERROR: atari800-3.1.0_1: failed to run do_configure() at line 7.
```
and fail.

**Comment by bougyman** [comment]
Now it builds but doesn't install :(.

```
=> atari800-3.1.0_1: running pre-install hook: 00-lib32 ...
=> atari800-3.1.0_1: running pre-install hook: 02-script-wrapper ...
=> atari800-3.1.0_1: running do_install ...
mkdir -p /destdir/arm-linux-gnueabihf7/atari800-3.1.0/usr/bin /destdir/arm-linux-gnueabihf7/atari800-3.1.0/usr/share/man/man1 /destdir/arm-linux-gnueabihf7/atari800-3.1.0/usr/share/doc/atari800
/usr/bin/install -c -s atari800 /destdir/arm-linux-gnueabihf7/atari800-3.1.0/usr/bin/atari800
strip: Unable to recognise the format of the input file `/destdir/arm-linux-gnueabihf7/atari800-3.1.0/usr/bin/atari800'
/usr/bin/install: strip process terminated abnormally
Makefile:141: recipe for target 'install' failed
make: *** [install] Error 1
=> ERROR: atari800-3.1.0_1: failed to run do_install() at line 20.
```

This may have to do with hostmakedepends vs makedepends.

File: srcpkgs/atari800/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+pkgname="atari800"
+version="3.1.0"
+revision=1
+build_style=gnu-configure
+short_desc="Emulator of Atari 800/800XL/130XE/5200 with various extensions"
+maintainer="Jakub Skrzypnik <jot.skrzyp@gmail.com>"
+depends="SDL SDL_mixer SDL_ttf SDL_image SDL_sound SDL_net ncurses  readline"
```
Comment (xtraeme) [inline srcpkgs/atari800/template]: This line is completely wrong...

File: srcpkgs/atari800/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+makedepends="SDL-devel SDL_mixer-devel SDL_ttf-devel SDL_image-devel SDL_sound-devel SDL_net-devel ncurses-devel readline-devel"
```
Comment (xtraeme) [inline srcpkgs/atari800/template]: SDL-devel is redundant because it's a dependency of the other SDL_foo pkgs.

File: srcpkgs/atari800/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
+pkgname="atari800"
+version="3.1.0"
+revision=1
+build_style=gnu-configure
+short_desc="Emulator of Atari 800/800XL/130XE/5200 with various extensions"
+maintainer="Jakub Skrzypnik <jot.skrzyp@gmail.com>"
+makedepends="SDL-devel SDL_mixer-devel SDL_ttf-devel SDL_image-devel
```
Comment (xtraeme) [inline srcpkgs/atari800/template]: SDL-devel is still redundant :-)

File: srcpkgs/atari800/template [hunk 4]
```diff
@@ -0,0 +1,14 @@
+pkgname="atari800"
+version="3.1.0"
+revision=1
+build_style=gnu-configure
+short_desc="Emulator of Atari 800/800XL/130XE/5200 with various extensions"
+maintainer="Jakub Skrzypnik <jot.skrzyp@gmail.com>"
+makedepends="SDL-devel SDL_mixer-devel SDL_ttf-devel SDL_image-devel 
+SDL_sound-devel SDL_net-devel ncurses-devel readline-devel MesaLib-devel"
```
Comment (xtraeme) [inline srcpkgs/atari800/template]: readline-devel depends on ncurses-devel, you can remove ncurses-devel as well.

File: srcpkgs/atari800/template [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+license="GPL-2"
+build_wrksrc="src/"
+homepage="http://atari800.sourceforge.net"
+configure_args="--disable-sdltest --with-video=sdl --with-opengl --with-sound=sdl --with-readline --with-x --enable-rserial --enable-rnetwork --enable-linuxjoystick --enable-ide --enable-voicebox --enable-pbi_xld --enable-clipsound --enable-consolesound --enable-volonlysound --enable-seriosound --enable-stereosound --enable-interpolatedsound --enable-nonlinear_mixing --enable-synchronized_sound --enable-pbi_mio --enable-pbi_bb --enable-eventrecording --enable-monitortrace --enable-monitorprofile --enable-monitorhints --enable-monitorbreakpoints --enable-monitorbreak --enable-monitorasm --enable-bufferedlog --enable-newcycleexact --enable-unalignedwords"
```
Comment (xtraeme) [inline srcpkgs/atari800/template]: Please avoid such long lines if possible.

**Pull request closed without merging**